pgctl/cli.py

```diff
@@ -238,6 +239,10 @@ def __change_state(self, state):
                     for service in self.all_services
                 )
 
+        if self._entering_debug:
+            # Do not post-stop when we are about to start a debug session
+            run_post_stop_hook = False
+
```

Comment:
I think ycp would still be ok if we nixed this actually. As long as pre-start is still called before debug actually begins.

I'm fine getting rid of this